learn/get-started/basics.md

```diff
@@ -1,13 +1,13 @@
 ---
 Order: 1
 Area: getstarted
-TOCTitle: Get started
+TOCTitle: Set Up
 ContentId: d8ebd3c0-ff4a-4e27-aba2-3acb9e17fff4
-PageTitle:  Get started with Visual Studio Code
+PageTitle:  Setting up Visual Studio Code
 DateApproved: 10/22/2020
 MetaDescription: Download and learn the basics of Visual Studio Code.
 ---
-# Get started with Visual Studio Code
+# Setting up Visual Studio Code
```

Comment:
Current Microsoft style guides try to limit gerunds in titles.
Would you be OK if I changed this to "Set up Visual Studio Code"?
I'd also say "Install Visual Studio Code extensions".
This matches "Customize Visual Studio Code"

Reply:
We chatted so I'll go ahead and merge this and make the changes above.